examples/model_selection/plot_tuned_decision_threshold.py

```diff
@@ -70,29 +70,46 @@
 # repetitions of 5-fold cross-validation.
 import pandas as pd
 
+from sklearn.metrics import make_scorer, recall_score
 from sklearn.model_selection import RepeatedStratifiedKFold, cross_validate
 
-scoring = ["accuracy", "balanced_accuracy"]
+scoring = {
+    "accuracy": "accuracy",
+    "balanced_accuracy": "balanced_accuracy",
+    "recall": make_scorer(recall_score, pos_label=pos_label),
+}
+
 cv_scores = [
     "train_accuracy",
     "test_accuracy",
     "train_balanced_accuracy",
     "test_balanced_accuracy",
+    "train_recall",
+    "test_recall",
 ]
+
 cv = RepeatedStratifiedKFold(n_splits=5, n_repeats=10, random_state=42)
-cv_results_vanilla_model = pd.DataFrame(
-    cross_validate(
-        model,
-        data,
-        target,
-        scoring=scoring,
-        cv=cv,
-        return_train_score=True,
-        return_estimator=True,
+
+
+def run_model_cv_results(model, cv):
+    cv_results_model = pd.DataFrame(
+        cross_validate(
+            model,
+            data,
+            target,
+            scoring=scoring,
+            cv=cv,
+            return_train_score=True,
+            return_estimator=True,
+        )
```

Comment:
This a bit counter intuitive and I would not recommend in production script but for our example, I would avoid to define a function and explicitly call the `pd.DataFrame` and `cross_validate`. The reason is that it avoids the reader to scroll up-down.

One way that we can minimize the call is to store some of the parameters that are set across the experiment:

```python
params = {
    "scoring": scoring, "cv": cv, "return_train_score": True, "return_estimator": True
}
cv_results_vanilla_model = pd.DataFrame(cross_validate(model, data, target, **params))
cv_results_vanilla_model[cv_scores].aggregate(["mean", "std"]).T
```

Then, we can reuse `params`